torch_xla/csrc/dl_convertor.cpp

```diff
@@ -0,0 +1,341 @@
+#include "torch_xla/csrc/dl_convertor.h"
+
+#include <ATen/DLConvertor.h>
+
+#include "absl/types/span.h"
+#include "torch_xla/csrc/aten_xla_bridge.h"
+#include "torch_xla/csrc/ops/device_data.h"
+#include "torch_xla/csrc/runtime/computation_client.h"
+#include "torch_xla/csrc/runtime/debug_macros.h"
+#include "torch_xla/csrc/runtime/pjrt_computation_client.h"
+#include "torch_xla/csrc/runtime/runtime.h"
+#include "torch_xla/csrc/runtime/tf_logging.h"
+#include "torch_xla/csrc/tensor.h"
+#include "torch_xla/csrc/tensor_util.h"
+#include "torch_xla/csrc/unwrap_data.h"
+#include "xla/pjrt/pjrt_client.h"
+#include "xla/pjrt/pjrt_future.h"
+#include "xla/pjrt/pjrt_layout.h"
+#include "xla/status.h"
+
+namespace torch_xla {
```

Comment:
Just checking: the only thing in this file that you copied from XLA was the function `DLDataTypeToPrimitiveType`, correct?

Reply:
There are a few relevant functions, such as PrimitiveTypeToDLDataType, DLDeviceForDevice, DeviceForDLDevice, StridesToLayout

Reply:
Could add a comment to all copied functions, so that we can easily keep track of?

Reply:
Added a comment to each function.